ami/ml/orchestration/async_job_state.py

```
@@ -182,6 +194,60 @@ def update_state(
+            sizes: list[str] = []
+            for key in keys:
+                if key == self._total_key:
+                    sizes.append(f"{key}=<str>")
+                    continue
+                try:
+                    sizes.append(f"{key}=SCARD:{redis.scard(key)}")
+                except RedisError:
+                    sizes.append(f"{key}=<err>")
+            keys_summary = ", ".join(sizes) if sizes else "<none>"
+            return f"redis db{db}: keys_for_job={keys_summary}"
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)